backend/.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
       - id: pretty-format-json
 
   - repo: https://github.com/psf/black
-    rev: 21.9b0
+    rev: 22.3.0
     hooks:
```

Comment:
how about this? will something break if we merge this? I can see it is a config file.

Reply:
Yeah I just updated the version of black to the latest version to take care of some of the issues with I was experiencing with the old version following a suggestion by the author on stackoverflow.